### PR TITLE
Removed nav-section dropdowns for items without children

### PR DESCRIPTION
**Pull request by inuwa**
## Done

Dropdowns were being created for nav sections without children on small screens. Now the nav sections do not create an accordion for nav section heading without children. 
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Minimise home screen and look at footer the links after the sector section should not be in accordion
format.

## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4140
## Screenshots

image

### CODE REVIEW

**Comment by anasereijo** [comment]
When we load the page Sectors dropdown is already opened and should be closed.
It also should have the same styling as the other dropdowns (for eg Download).

screenshot 2018-10-30 at 10 44 06

**Comment by inuwa** [comment]
> When we load the page Sectors dropdown is already opened and should be closed.
> It also should have the same styling as the other dropdowns (for eg Download).
> 
> screenshot 2018-10-30 at 10 44 06

Hi @anasereijo This is actually what the drop menu should look like minimised. What you have 👆 is the dropdown closed above.
[image](https://user-images.githubusercontent.com/43535482/47723433-6c748180-dc4c-11e8-9a7c-57cd00ba143b.png)

**Comment by anasereijo** [comment]
@inuwa as we discussed can we please make the following changes:

- match Sectors style and left-padding to the other dropdowns
- remove the arrow from the non-dropdown links

see design below:
the leading operating system for pcs iot devices servers and the cloud ubuntu

**Comment by inuwa** [comment]
@anasereijo it is done now.👊 Cheers

**Comment by anasereijo** [comment]
Hi @inuwa,

it's looking much better but now the drop-down items have been moved to the right as you can see below:

screenshot 2018-10-31 at 13 46 02

how it should look like (please check online version):

screenshot 2018-10-31 at 13 46 24

**Comment by inuwa** [comment]
@anasereijo yes it is done. Hopefully it gets a pass now. Cheers😁

**Comment by anasereijo** [comment]
@inuwa looking good 👍

File: static/sass/_pattern_footer.scss [hunk 1]
```diff
@@ -1,268 +1,302 @@
 // Footer pattern styling
 
 @mixin footer-title {
-  color: $color-mid-dark;
-  cursor: pointer;
-  line-height: 1.5;
-  margin-bottom: 0;
+	color: $color-mid-dark;
```
Comment (deadlight) [inline static/sass/_pattern_footer.scss]: This file has had spaces converted to tabs. Check your editor settings and revert to spaces, please.

File: static/sass/_pattern_footer.scss [hunk 2]
```diff
@@ -1,268 +1,302 @@
 // Footer pattern styling
 
 @mixin footer-title {
-  color: $color-mid-dark;
-  cursor: pointer;
-  line-height: 1.5;
-  margin-bottom: 0;
+    color: $color-mid-dark;
```
Comment (deadlight) [inline static/sass/_pattern_footer.scss]: Sorry... but it should be 2 spaces per indent, not 4!

Comment (inuwa) [reply static/sass/_pattern_footer.scss]: Hi, @deadlight no worries, I have reduced the indents to 2 spaces now. Cheers

**Pull request merged**